src/__test__/compile/importPackage.js

```diff
@@ -1,4 +1,4 @@
-import jszip from 'jszip';
```

Comment:
JSZip is big, and takes nearly a second to compile on my machine. Switching to a smaller package for this test case shaves that much off each test run.